### ISSUE TITLE
Accessibility: MAS36: Keyboard: Links are not accessible with keyboard.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#31]
#A11yMAS
#MAS36
#Wipro
#vscode-website
#FireFox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
1.Open https://code.visualstudio.com/ in edge.
2.Navigate to the ".zip" under windows download and ".tar.gz" under .deb download below the "Get Started Now" in middle of the page
3.Try to access the links.

**Actual Result:**
Unable to access the ".zip" and ".tar.gz" links with keyboard in middle of the page.

**Expected Result:**
The user should be able  to access the all the interactive elements available in the page.
Note: As the alt text is empty , the narrator reads the rating stars as "image"

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may find it difficult while navigating with the tab to use the Application. 

**Screenshots:**
[unable to access the links.zip](https://github.com/wiproaccess/vscode-website/files/689111/unable.to.access.the.links.zip)

**Comment by chrisdias** [comment]
Fixed with https://github.com/Microsoft/vscode-website/commit/6efd7c2db5268d27dd8c690eb8a501420fe345ad

and

https://github.com/Microsoft/vscode-website/commit/e5666cde8df5932957fc3dd1af31f88d27faaee2
